### ISSUE TITLE
	positive regulation of mitotic sister chromatid arm separation

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#24141]
there is

positive regulation of mitotic sister chromatid arm separation

but no 

positive regulation of mitotic sister chromatid separation

(maybe we can repurpose the arm separation)?
in mitosis there is not difference in process for arms and centromeres?

**Comment by pgaudet** [comment]
These terms do exist as far as I can tell. Do you want to merge

- [ ] GO: 1990891 'mitotic sister chromatid arm separation' into GO:0051306 'mitotic sister chromatid separation
- [ ] GO:1905822 'regulation of mitotic sister chromatid arm separation'  into GO:0010965 'regulation of mitotic sister chromatid separation'
- [ ] GO:1905823 'negative regulation of mitotic sister chromatid arm separation' into GO:2000816 'negative regulation of mitotic sister chromatid separation' <http://purl.obolibrary.org/obo/>
- [ ] GO:1905824 'positive regulation of mitotic sister chromatid arm separation into GO:1901970 'positive regulation of mitotic sister chromatid separation' 



? 

Thanks, Pascale

**Comment by ValWood** [comment]
Agreed. I could not find "positive regulation of mitotic sister chromatid separation" but it is there...

merge all.

**Comment by raymond91125** [comment]
Is the difference between arms and centromere not valid anymore?
PMID:11081627  (2000) Two distinct pathways remove mammalian cohesin from chromosome arms in prophase and from centromeres in anaphase

**Comment by ValWood** [comment]
Well spotted. There are many other issues here too.

We don't currently have a separate term for 
"mitotic sister chromatid centromere separation" I think everyone has been using the general "mitotic sister chromatid centromere separation".

the part which controls "chromosome segregation at anaphase"  (i.e. the cleavage of cohesion by separase).
I think this is what we are trying to describe by this term

The events which occur in promeataphase/metaphase  are  "cohesin unloading" events  which we recently decided were single step, we previously said we would model this as "negative regulation of cohesion" 

Then we have another bunch of cohesion terms

GO:0099404    mitotic sister chromatid cohesion, telomeric | is_a
GO:0071962    mitotic sister chromatid cohesion, centromeric | is_a
GO:0034088    maintenance of mitotic sister chromatid cohesion | part_of
GO:0071961    mitotic sister chromatid cohesion, arms | is_a
GO:0034087    establishment of mitotic sister chromatid cohesion
GO:0034184    positive regulation of maintenance of mitotic sister chromatid cohesion | positively_regulates
GO:0099403    maintenance of mitotic sister chromatid cohesion, telomeric | is_a
GO:0071959    maintenance of mitotic sister chromatid cohesion, arms | is_a
GO:0034182    regulation of maintenance of mitotic sister chromatid cohesion | regulates
GO:0071960    maintenance of mitotic sister chromatid cohesion, centromeric | is_a
GO:0034183    negative regulation of maintenance of mitotic sister chromatid cohesion
and more.....

It might be better to begin by  collecting all of the terms  related to cohesion and separation and reviewing together 
Some of these  cohesion vs separation terms are saying the same thing. 

We need to work out which ones are required to describe the specific regulation pathways.

Also that the spindle checkpoint signalling is the pathway which controls cleavage of cohesion, so 
GO:0034182    regulation of maintenance of mitotic sister chromatid cohesion 
when referring to the centromere is really describing the spindle checkpoint pathway.

The only certain thing here is that we need fewer terms....

**Comment by ValWood** [comment]
I have been working on a Noctua model for this process:
https://github.com/pombase/curation/issues/3774
I still need to model the protection of centromeric cohesion at mitotic metaphase, but essentially I don't think we need all of these terms at all.

I  will make a proposal to run by @colinlog and the rest of the GO editors (not much has changed since the previous discussion, but I propose some relabelling of the remaining terms)
